<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DatabaseWindow</class>
 <widget class="QWidget" name="DatabaseWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>750</width>
    <height>615</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>750</width>
    <height>615</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>750</width>
    <height>615</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MySQL</string>
  </property>
  <widget class="QPlainTextEdit" name="te_output">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>110</y>
     <width>191</width>
     <height>191</height>
    </rect>
   </property>
   <property name="cursor" stdset="0">
    <cursorShape>IBeamCursor</cursorShape>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_connect">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>20</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>执行</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_showAllDb">
   <property name="geometry">
    <rect>
     <x>319</x>
     <y>110</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>显示所有数据库</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>25</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>使用命令</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="te_input">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>15</y>
     <width>481</width>
     <height>71</height>
    </rect>
   </property>
   <property name="cursor" stdset="0">
    <cursorShape>IBeamCursor</cursorShape>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>显示结果</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_showCurDb">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>160</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>显示当前数据库</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_openDb">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>110</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>切换数据库</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_showTable">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>210</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>显示数据表</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cb_db">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>110</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
  </widget>
  <widget class="QComboBox" name="cb_table">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>210</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
  </widget>
  <widget class="QLabel" name="lb_curDb">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>160</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>IBeamCursor</cursorShape>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_setTable">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>210</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>选择数据表</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>270</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>当前数据表</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lb_curTable">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>270</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>IBeamCursor</cursorShape>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>310</y>
     <width>581</width>
     <height>251</height>
    </rect>
   </property>
   <property name="cursor" stdset="0">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>570</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>序号范围</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_idStart">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>570</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_idEnd">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>570</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>570</y>
     <width>21</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>-</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_inquire">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>570</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>查询</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_display">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>570</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>显示全表</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>601</x>
     <y>321</y>
     <width>139</width>
     <height>241</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="pb_showTableData">
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>显示数据表内容</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pb_commit">
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>提交修改</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pb_repeal">
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>撤销修改</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pb_record">
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>添加记录</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pb_remove">
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>删除选中行</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>te_input</tabstop>
  <tabstop>pb_connect</tabstop>
  <tabstop>te_output</tabstop>
  <tabstop>pb_showAllDb</tabstop>
  <tabstop>cb_db</tabstop>
  <tabstop>pb_openDb</tabstop>
  <tabstop>pb_showCurDb</tabstop>
  <tabstop>pb_showTable</tabstop>
  <tabstop>cb_table</tabstop>
  <tabstop>pb_setTable</tabstop>
  <tabstop>tableView</tabstop>
  <tabstop>pb_showTableData</tabstop>
  <tabstop>pb_commit</tabstop>
  <tabstop>pb_repeal</tabstop>
  <tabstop>pb_record</tabstop>
  <tabstop>pb_remove</tabstop>
  <tabstop>le_idStart</tabstop>
  <tabstop>le_idEnd</tabstop>
  <tabstop>pb_inquire</tabstop>
  <tabstop>pb_display</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
